rest/client/interface.go

```diff
@@ -95,6 +95,9 @@ type Communicator interface {
 	GetJSONData(context.Context, TaskData, string, string, string) ([]byte, error)
 	GetJSONHistory(context.Context, TaskData, bool, string, string) ([]byte, error)
 
+	// GenerateTasks posts new tasks for the `generate.tasks` command.
+	GenerateTasks(context.Context, TaskData, interface{}) error
```

Comment:
can you put the third argument as a model in apimodels? 

or actually have it be bytes, so that the server can just parse the payload you send

Reply:
done